draft-ladd-privacypass-bbs.md

```diff
@@ -224,8 +224,18 @@ The position of a revealed attribute, as well as the number of unrevealed attrib
 
 # IANA Considerations
 
-This document has no IANA actions.
-
+We would like IANA to add to the Privacy Pass Token Type Registry the following registration:
+
+Value: IANA picks
+Name: BBS Token
+Token structure: As in Token Generation Section
+Token Key Encoding: TODO
+Publicly Verifiable: Y
+Public Metadata: Y
+Private Metadata: N
+Nk: Indefinite
+NiD: 32? TODO: check this
```

Comment:
We need to add the Issuer configuration details to remove this TODO, but I do believe it will be 32 as well.